inc/class-uds-notification-banner.php

```diff
@@ -112,17 +112,25 @@ public function widget( $args, $instance ) {
 
 		// We now have all the variables we need to construct the actual banner.
 		echo <<<EOS
-		<div class="banner banner-$color" role="banner">
-		<div class="banner-icon">
-			<span title="Banner" class="fa fa-icon fa-$icon"></span>
-		</div>
-			<div class="banner-content">
-				<h3>$title</h3>
-				$body
-			</div>
-			$button_block
-			<div class="banner-close">
-				<button type="button" class="btn btn-circle btn-circle-alt-white close" aria-label="Close" onclick="event.target.parentNode.parentNode.style.display='none';">x</button>
+		<div class="container-fluid banner-$color">
+			<div class="container">
```

Comment:
I noticed that you added this **container-fluid** with the same background color as the the banner, so now we have a wider banner's background but the content and "x" button are still within the container width. 
Is this how banners should look like?

Reply:
I think so, based on the examples in the XD file. It even keeps the "nothing lines up at all" aesthetic of the original!

![banners](https://user-images.githubusercontent.com/31013359/115474744-f46eed00-a1f2-11eb-86f2-4e5163838065.png)